test/compare_test.go

```diff
@@ -0,0 +1,80 @@
+package test
+
+import (
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/vkuznet/transfer2go/core"
+)
+
+// Compare 1k files of source with the same 1k files of destination
+func TestCompareThousand(t *testing.T) {
```

Comment:
I would suggest either to delete this test since you have identical TenThousand one or make test to accept optional parameter for number of catalog entries, e.g. 1k or 10k or whatever. So far TestCompareThousand and TestCompareTenThousand are identical except one parameter in their body.